platform/admin/ui/src/main/webapp/js/views/Alerts.view.js

```diff
@@ -16,36 +16,71 @@
 define([
         'marionette',
         'text!templates/alerts.handlebars',
-        'icanhaz'
+        'icanhaz',
+        'jquery'
     ],
-    function (Marionette, alertsTemplate, ich) {
+    function (Marionette, alertsTemplate, ich, $) {
 
         ich.addTemplate('alertsTemplate', alertsTemplate);
 
         var AlertsView = {};
 
+        var dismissUrl = '/admin/jolokia/exec/org.codice.ddf.ui.admin.api:type=AdminAlertMBean/dismissAlert';
+
         AlertsView.View = Marionette.ItemView.extend({
             template: 'alertsTemplate',
-            tagName: 'table',
             events: {
                 'shown.bs.collapse': 'toggleDetailsMsg',
-                'hidden.bs.collapse': 'toggleDetailsMsg'
+                'hidden.bs.collapse': 'toggleDetailsMsg',
+                'click .dismiss': 'dismissAlert'
             },
             modelEvents: {
                 'change': 'render'
             },
             /*jshint -W030 */
             toggleDetailsMsg: function () {
                 var model = this.model;
-                model.get('button') === 'Show' ? model.set('button', 'Hide') : model.set('button', 'Show');
+                model.get('details-button-action') === 'Show' ? model.set('details-button-action', 'Hide') : model.set('details-button-action', 'Show');
                 model.get('collapse') === 'out' ? model.set('collapse', 'in') : model.set('collapse', 'out');
             },
             serializeData: function () {
                 var json = this.model.toJSON();
                 json.collapseId = 'alertCollapse_' + parseInt((Math.random() * Math.pow(2, 32)), 10);
+                switch (this.model.get('priority')) {
+                    case 2:
```

Comment:
❓  There's one priority not here, right? If someone uses that, what will happen?